README.rst

```diff
@@ -40,8 +40,10 @@
 .. image:: https://img.shields.io/github/license/inveniosoftware/invenio-records-rest.svg
         :target: https://github.com/inveniosoftware/invenio-records-rest/blob/master/LICENSE
 
-
 REST API for invenio-records.
 
-* Free software: GPLv2 license
-* Documentation: https://invenio-records-rest.readthedocs.io/
+*This is an experimental developer preview release.*
```

Comment:
This line can be removed. This is not clear in the checklist that it has to be removed, when we go to beta release.